smithy-jsonschema/src/test/java/software/amazon/smithy/jsonschema/JsonSchemaConverterTest.java

```diff
@@ -551,6 +552,25 @@ public void supportsMapPatternPropertiesWithDefaultPattern() {
         assertThat(schema.getPatternProperties().get(".+").getType().get(), equalTo("string"));
     }
 
+    @Test
+    public void sensitiveTraitHasNotImpact() {
```

Comment:
One of these strings shouldn't have the trait applied.

Reply:
haha, copy paste error. fixing test name too.